tests/unit/FormCheckbox.spec.js

```diff
@@ -0,0 +1,81 @@
+import { shallowMount } from '@vue/test-utils'
+import FormCheckbox from '../../src/components/FormCheckbox.vue';
+
+describe('FormCheckbox',() => {
+    let wrapper;
+    const label = 'Accept Terms';
+    beforeEach(() => {
+        wrapper = shallowMount(FormCheckbox, {
+            propsData: {
+                label: label
+            }
+        });
+    });
+
+    it('renders the component', () => {
+        expect(wrapper.find('input').attributes().type).toBe('checkbox');
+        expect(wrapper.html()).toContain(label);
+    });
+
+    it('renders all configured properties', () => {
+        const helper = "This is some text";
+        const name = "checkboxName";
+        const error = "This field has an error";
+        const toggle = true;
+        const initiallyChecked = true;
+        const readOnly = false;
+
+        wrapper.setProps({
+            label: label,
+            name: name,
+            helper: helper,
+            toggle: toggle,
+            error: error,
+            initiallyChecked: initiallyChecked,
+            disabled: readOnly
+        });
```

Comment:
I wonder if this is the correct way to test these? I have two concerns with it:
 * It tests _changing_ the props, but not how the component behaves when it initially renders with these props already set.
 * Calling `setProps` uses an API that will never be available to any user in the app (it changes the props in a programmatic way that I think should be reversed for special cases).

To avoid these two issues, instead of using a `beforeEach` hook, you can create a helper method that you could use like this:
```js
const wrapper = getWrapper(initialProps);
```